docs/swagger.yaml

```diff
@@ -197,6 +207,57 @@ components:
       schema:
         type: string
         example: "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1Ni..."
+    Merchant-Serial-Number:
```

Comment:
This standard header in our APIs make less sense for this API because you query using `ledgerID`, and potentially for many Merchant-Serial-Number at the same time. So I suggest we remove this headers but leave the rest.